.changeset/gold-horses-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fix: check when popup is closed to proceed with login
```

Comment:
changesets are for release notes and should user-friendly. Check it out the guidelines for it
https://developer.rocket.chat/contribute-to-rocket.chat/ways-to-contribute/developing/development-workflow#adding-changeset-to-your-pull-request

```suggestion
---
"@rocket.chat/meteor": patch
---

Fixed CAS login after popup closes
```

Reply:
thank you for the notice